### PR TITLE
Fix ugly padding

### PR DESCRIPTION
**Pull request by jorenbroekema**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,3 +75,14 @@ This is recommended, because it avoids deduplication issues.
...
 In Backlight, this is taken care of for you on the backend, and you won't have to worry about it.
 If you use `starter-simba` outside of Backlight, you will need to ensure that deduplication is handled, otherwise you will run into breaking bugs.
+
+### !important in ::slotted selectors
+
+Right now, as per W3C specs, ::slotted CSS specificity is just that of a pseudo element.
+Unlike :host, it does not include the specificity of the passed argument selector.
+
+This means that almost any selector from outside that targets the slotted element will override your ::slotted CSS.
+This is awful, [there's an open issue](https://github.com/w3c/csswg-drafts/issues/6466) but hasn't gained much traction.
```
Comment (bashmish) [inline README.md:85]: I'd say `This is not ideal/optimal...` or `Meanwhile a group of people tries to improve the spec...`

Comment (jorenbroekema) [reply README.md]: Rephrased it

**Comment by bashmish** [comment]
Good, let's see where it's all going, but for now it seems to be the only acceptable solution! 👍

File: studio.config.json [hunk 2]
```diff
@@ -28,9 +37,28 @@
...
+        ]
+      ],
+      [
+        "re-exports-only",
+        [
+          "core",
+          "localize",
+          "overlays"
+        ]
+      ]
```
[the remark this answers is not in the document]
Comment (jorenbroekema) [reply studio.config.json:61]: Formatting rules between local editor and backlight editor. They're using different versions of Prettier I think..

**Pull request merged**